pom.xml

```diff
@@ -53,17 +53,17 @@
     </developers>
 
     <properties>
-        <jenkins.version>1.609.3</jenkins.version>
-        <!--<jenkins.version>2.5-SNAPSHOT</jenkins.version>-->
-        <java.level>6</java.level>
+        <!-- <jenkins.version>2.14</jenkins.version>  First version with auto-closeable ACLs / 2016 -->
+        <jenkins.version>2.60.1</jenkins.version> <!-- First LTS for java 8 only / 2017 -->
+        <java.level>8</java.level>
         <no-test-jar>false</no-test-jar>
         <findbugs.failOnError>false</findbugs.failOnError>
         <concurrency>1C</concurrency>
         <powermock.version>1.6.2</powermock.version>
         <checkstyle.version>2.13</checkstyle.version>
         <mockito.version>1.10.19</mockito.version>
-        <workflow.version>1.14</workflow.version>
-        <workflow.version.test>1.14.1-beta-1</workflow.version.test>
+        <workflow.version>1.14.2</workflow.version>
```

Comment:
I believe we need to bump this to some 2.x version of workflow and split up the versions to whatever latest that supports core 2.60. 1.14 of workflow is ancient.

Reply:
All right, I check that.

Reply:
I need some advice for this point. If I use workflow-step-api plugin versions targeted to java 8 (core 2.60), I get the last version of this plugin. I can't wonder if it's a problem or not. If we do that, gerrit-trigger-plugin users will be forced to use the last workflow-step-api version to update gerrit-trigger-plugin ?

May be not an issue, but I can't guess ...

Version of workflow-step-api needing java 8 / jenkins core 2.60 : 2.18 (january 2019) (runtime optional)

Reply:
workflow-aggregator needed for some tests introduce a lot of dependencies problems (maven enforcer). Do you think there is a good way to replace it for test, or we really need to deal with all depencies management ? Maybe we could get an example of dependencies management from other project using it ?

Or ... keep this versions of workflow plugins ? Since we do need new versions features, and their versions are drived Jenkins plugin management, it's an important issue ?

Reply:
We don't have to go with the latest bleeding edge versions but something 2.x so that debugging etc gets easier to do.
Also as of workflow suite 2.0 there is no longer a mono repo of all the basics of workflow instead each plugin in the suite has it's own release cycle so we can't use the agregator any more.
See #390 for examples.